### PR TITLE
Dev

### PR DESCRIPTION
**Pull request by Nkaka23dev**

### CODE REVIEW

File: responsive.css [hunk 1]
```diff
@@ -0,0 +1,301 @@
...
+        letter-spacing: 0.2px;
+
+        /* second-text-color */
+
+        color: #737373;
+    }
+
+    /* ================================================SUB-HEADER FOR SMALL DEVICES START HERE ==================== */
+    .sub-h-container {
+        margin-top: 4rem;
```
Comment (NNesta) [inline responsive.css:74]: You should name using descriptive name

File: responsive.css [hunk 2]
```diff
@@ -0,0 +1,301 @@
...
+    }
+
+    .p-text {
```
Comment (NNesta) [inline responsive.css:77]: You should name using descriptive name

File: responsive.css [hunk 3]
```diff
@@ -0,0 +1,301 @@
...
+        width: 100%;
+        height: 2rem;
+    }
+
+    .form-input__control {
+        padding: 0.5rem 0.3rem;
+        width: 100%;
+    }
+
+    .form_text {
```
Comment (NNesta) [inline responsive.css]: This is not BEM

File: responsive.css [hunk 4]
```diff
@@ -0,0 +1,301 @@
...
+    }
+
+    .bg-image-wrapper {
+        height: 105vh;
+
+    }
+
+    .btn--orange {
+        width: 100%;
+        margin-top: 1.5rem;
```
Comment (NNesta) [inline responsive.css:209]: This modifier does not modify what it is named for

**Review by NNesta — CHANGES_REQUESTED** [review]

File: responsive.css [hunk 5]
```diff
@@ -0,0 +1,301 @@
...
+    }
+    .v-container__img {
+      width: 70%;
+      height: 70%;
+    } 
+    .v-container__filter {
+        width: 100%;
+        height: 50vh; 
+    }
+    /* .v-container__btn {
```
Comment (chrisostome01) [inline responsive.css]: You can remove this commented style

File: responsive.css [hunk 6]
```diff
@@ -0,0 +1,301 @@
...
+    }
+
+    .form-container__sub-heading {
+        font-weight: 700;
+        text-align: center;
+        font-size: 13px;
+        line-height: 24px;
+        /* identical to box height, or 150% */
+        letter-spacing: 0.1px;
+
```
Comment (chrisostome01) [inline responsive.css]: Try to avoid unnecessary spaces in your CSS,

File: main.css [hunk 7]
```diff
@@ -0,0 +1,525 @@
...
+    justify-content: space-between;
+    gap: 10rem;
+}
+
+.navbar-container__items {
+    display: flex;
+    gap: 1rem;
+}
+
+.items {
```
Comment (chrisostome01) [inline main.css:52]: May you try to place this Item after navbar, for better structure

**Review by chrisostome01 — CHANGES_REQUESTED** [review]
Well done @Nkaka23dev, I left some comments you can take look at it. 👍👌

File: index.html [hunk 8]
```diff
@@ -184,20 +184,77 @@ <h4 class="mech-section__txt">Digital markerting</h4>
...
+                            <div class="client-card__detail">
+                                <p class="client-card__names">Regina Miles</p>
+                                <p class="client-card__profession">Designer</p>
+                            </div>
+                        </div>
+                    </div>
                 </div>
                 <div>
-                    <img width="300px" height="200px" src="./images/card-item.png" alt="">
+                    <div class="client-card">
```
Comment (chrisostome01) [inline index.html:231]: Well done @Nkaka23dev, now i think it is easy, to modify this card's content

**Review by chrisostome01 — APPROVED** [review]
Well done

**Review by NNesta — APPROVED** [review]
Nice done @Nkaka23dev

File: main.css [hunk 9]
```diff
@@ -0,0 +1,557 @@
...
+    display: flex;
+    gap: 10px;
+    padding: 0.5rem 0rem;
+}
+
+.form-input__control {
+    padding: 0.5rem 1.5rem;
+}
+
+input[type=submit]:hover {
```
Comment (karemarsy) [inline main.css]: this is not BEM

File: index.html [hunk 10]
```diff
@@ -1,12 +1,454 @@
...
+                </div>
+                <box-icon name='menu-alt-right'></box-icon>
+            </div>
+            <div class="navbar-container__icons-sm">
+                <box-icon name='search'></box-icon>
+                <box-icon name='cart-alt'></box-icon>
+                <box-icon name='menu-alt-right'></box-icon>
+            </div>
+        </nav>
+        <div class="navbar-container__menu-items-sm">
```
Comment (karemarsy) [inline index.html]: `navbar-container__menu-items-sm` is positioned outside the Block `(navbar-container)`.

File: index.html [hunk 11]
```diff
@@ -1,12 +1,454 @@
...
+            <div class="menu-sm">
+                <span class="navbar-container__item">Home</span>
```
Comment (karemarsy) [inline index.html]: `navbar-container__item` is positioned outside the Block `(navbar-container)`.

File: index.html [hunk 12]
```diff
@@ -1,12 +1,454 @@
...
+            </div>
+        </div>
+    </header>
+    <!-- ==================================================END OF MENU SECTIPON =============================== -->
+    <div class="container">
+        <div class="container__content">
+            <div class="page-text">
+                <p class="p-text">Experts are here solve your business problem</p>
+            </div>
+            <div class="page-text__sub-txt">
```
Comment (karemarsy) [inline index.html]: `page-text__sub-txt` is positioned outside the Block `(page-text)`.

File: index.html [hunk 13]
```diff
@@ -1,12 +1,454 @@
...
+            </div>
+        </div>
+        <!-- =========================================================================================== -->
+        <div class="container container--sm">
+            <div class="container__content">
+                <div class="page-text">
+                    <p class="p-text p-text--sm p-text-sm--normal">We are providing best <br> <span>business
+                            services.</span> </p>
+                </div>
+                <div class="page-text__sub-txt page-text__sub-txt--sm">
```
Comment (karemarsy) [inline index.html]: `page-text__sub-txt` is positioned outside the Block `(page-text).` and also
`page-text__sub-txt--sm `is positioned outside the Block `(page-text)`.

File: index.html [hunk 14]
```diff
@@ -1,12 +1,454 @@
...
+                            consectetur adipisicing</p>
+                    </div>
+                </div>
+            </div>
+        </div>
+        <!-- =========================================================================================== -->
+        <div class="container container--sm">
+            <div class="container__content">
+                <div class="page-text">
+                    <p class="p-text p-text--sm p-text-sm--normal">We are providing best <br> <span>business
```
Comment (karemarsy) [inline index.html]: `p-text-sm--normal` is included without it's default Element `p-text-sm.`

File: index.html [hunk 15]
```diff
@@ -1,12 +1,454 @@
...
+        <div class="v-container">
+            <img class="v-container__img" src="./images/media bg-cover.png" alt="">
+            <img class="v-container__filter" src="./images/filter.png" alt="">
+            <img class="v-container__btn" src="./images/rouded img.png" alt="">
+        </div>
+        <!-- =================================================================================================== -->
+        <div class="container container--sm">
+            <div class="container__content">
+                <div class="page-text">
+                    <p class="p-text p-text--sm p-text-sm--normal">What Clients says</span> </p>
```
Comment (karemarsy) [inline index.html]: `p-text-sm--normal` is included without it's default Element `p-text-sm.`

File: index.html [hunk 16]
```diff
@@ -1,12 +1,454 @@
...
+                    </div>
+                </div>
+                <!-- end of card one -->
+            </div>
+        </div>
+        <!-- ======================================================================================================= -->
+        <div class="container container--sm  container--bg-white">
+            <div class="container__content">
+                <div class="page-text">
+                    <p class="p-text p-text--sm p-text-sm--normal">Meet Our Team</p>
```
Comment (karemarsy) [inline index.html]: `p-text-sm--norma`l is included without it's default Element `p-text-sm.`

File: index.html [hunk 17]
```diff
@@ -1,12 +1,454 @@
...
+                </div>
+                <div class="page-text__sub-txt page-text__sub-txt--sm">
+                    <p>Problems trying to solve the conflict between<br> <span>the two realms of Classical Physics:
+                            Newtonians Mechanics </span> </p>
+                </div>
+            </div>
+        </div>
+        <!-- =================================================================================================== -->
+        <div class="card-container">
+            <div class="card__wrapper">
```
Comment (karemarsy) [inline index.html]: `card__wrapper` is positioned outside the Block `card`.

File: index.html [hunk 18]
```diff
@@ -1,12 +1,454 @@
...
+                    <p class="sub-title">The quick fox jumps over the lazy dog</p>
+                </div>
+                <div class="btn btn--orange btn--sm">Contact Us</div>
+            </div>
+        </div>
+        <!-- ========================================================================================== -->
+        <div class="footer">
+            <div class="footer__content">
+                <div>
+                    <h2 class="logo--white">KMarketer</h2>
```
Comment (karemarsy) [inline index.html]: `logo--white` is included without its default Element `logo.`

File: index.html [hunk 19]
```diff
@@ -1,12 +1,454 @@
...
+                        <p>Lorem, ipsum dolor.</p>
+                    </div>
+                </div>
+            </div>
+        </div>
+        <!-- =================================================================================================== -->
+        <div class="footer footer--colored">
+            <div class="footer__content">
+                <div>
+                    <h1 class="logo logo--white logo--sm">Made With Love By <br class="separator"> Figma Land All Right
```
Comment (karemarsy) [inline index.html:442]: `logo--white` is included without its default Element `logo.`

File: index.html [hunk 20]
```diff
@@ -1,12 +1,454 @@
...
+                    <box-icon color="#335BF5" type='logo' name='facebook-square'></box-icon>
+                    <box-icon color="#E51F5A" type='logo' color="#1BA0AB" name='instagram'></box-icon>
+                    <box-icon color="#21A6DF" type='logo' color="#1BA0AB" name='twitter'></box-icon>
+                    <box-icon color="red" type='logo' name='youtube'></box-icon>
+                </div>
+            </div>
+        </div>
+        <!-- =================================================================================================== -->
+        <div class="footer footer--big footer--start-placed-content">
+            <div class="footer__content footer-content--unbordered">
```
Comment (karemarsy) [inline index.html]: `footer-content--unbordered` is included without it's default Element `footer-content`.

File: main.css [hunk 21]
```diff
@@ -0,0 +1,550 @@
...
+    letter-spacing: 0.2px;
+    color: #252B42;
+}
+
+.form-input__select {
+    width: 13.4rem;
+    height: 2rem;
+}
+
+.col-75 {
```
Comment (chrisostome01) [inline main.css]: It would be better if rename this block with meaningful name

File: main.css [hunk 22]
```diff
@@ -0,0 +1,550 @@
...
+}
+
+.footer--colored {
+    height: 10vh;
+    overflow-y: hidden;
+    padding: 0rem 0rem;
+    background-color: #16697A;
+}
+
+.logo--sm {
```
Comment (chrisostome01) [inline main.css]: To structure your css it is better to put your block, block element, and modifier in the same places for instance you can have this `logo--sm` in the same place as where you defined your block

File: index.html [hunk 23]
```diff
@@ -1,12 +1,455 @@
...
+            </div>
+        </div>
+        <!-- =================================================================================================== -->
+        <div class="footer footer--big footer--start-placed-content">
+            <div class="footer__content footer__content--unbordered">
+                <div>
+                    <h4 class="footer-title">Company Info</h4>
+                    <div class="footer-content">
+                        <p c>About Us</p>
+                        <p>Carrier</p>
```
Comment (NNesta) [inline index.html]: remove that typo <p c>

File: index.html [hunk 24]
```diff
@@ -1,12 +1,455 @@
...
+                        <p>We are hiring</p>
+                        <p>Blog</p>
+                    </div>
+                </div>
+                <div>
+                    <h4 class="footer-title">Legal</h4>
+                    <div class="footer-content">
+                        <p c>About Us</p>
+                        <p>Carrier</p>
+                        <p>We are hiring</p>
```
Comment (NNesta) [inline index.html]: remove typo <p c>

**Review by NNesta — CHANGES_REQUESTED** [review]

**Review by chrisostome01 — CHANGES_REQUESTED** [review]
Well done @Nkaka23dev, I left some comments for you, try to work on them.

**Review by chrisostome01 — APPROVED** [review]
LGTM 👍👍

**Review by NNesta — APPROVED** [review]
LGTM

**Review by karemarsy — APPROVED** [review]

**Pull request merged**